elasticdl/python/master/servicer.py

```diff
@@ -19,6 +20,7 @@ def __init__(
         self._version = 0
 
         self._evaluation_service = evaluation_service
+        self._task_complete_times = [36000] * 20
```

Comment:
Is `20` the task count to calculate the average process time? If yes, it is more readable to use a constant such as 
`STATISTICS_TASK_COUNT = 20`

Reply:
Done